risc0/r0vm/Cargo.toml

```diff
@@ -35,6 +35,7 @@ opentelemetry-otlp = "0.30.0"
 opentelemetry_sdk = "0.30.0"
 reqwest = { version = "0.12", default-features = false }
 risc0-circuit-rv32im = { workspace = true }
+risc0-zkp = { workspace = true }
```

Comment:
Why is this needed? Can we forward whatever we need thru `risc0-zkvm` instead?

Reply:
I've done just that and re-exported `VerificationError` from `risc0-zkvm`